web/src/test/java/com/fitgoal/web/util/TestHelper.java

```diff
@@ -0,0 +1,15 @@
+package com.fitgoal.web.util;
+
+import io.dropwizard.db.DataSourceFactory;
+
+public class TestHelper {
```

Comment:
You have 3 classes with the same name what can make it painful to find the class you need :-)
![image](https://user-images.githubusercontent.com/9981628/68756101-3c5b0f00-0612-11ea-81ec-033173593f37.png)

Reply:
Renamed